cli_tools/gce_vm_image_import/importer/importer.go

```diff
@@ -150,12 +150,17 @@ func buildDaisyVars(translateWorkflowPath, imageName, sourceFile, sourceImage, f
 	if translateWorkflowPath != "" {
 		varMap["translate_workflow"] = translateWorkflowPath
 		varMap["install_gce_packages"] = strconv.FormatBool(!noGuestEnvironment)
+		varMap["isWindows"] = strconv.FormatBool(strings.Contains(translateWorkflowPath, "windows"))
 	}
 	if sourceFile != "" {
 		varMap["source_disk_file"] = sourceFile
 	}
 	if sourceImage != "" {
-		varMap["source_image"] = fmt.Sprintf("global/images/%v", sourceImage)
+		if strings.Contains(sourceImage, "global/images") {
```

Comment:
I have a similar change to do this. Is this required by what you are doing? It looks like different thing

Reply:
I guess this is due to Eric merging your changes? Although your changes are not yet merged. It's weird

Reply:
Yeah, it was a bug that I found when running the command below:

```
go run compute-image-tools/cli_tools/gce_vm_image_import/main.go \
  -client_id=edens-test \
  -source_image=projects/ajackura-dev/global/images/customer-srv203-issue886 \
  -os=windows-2008r2 \
  -image_name=guest-os-test-7 \
  -project=edens-test
```

Reply:
It's basically the same thing Tony worked on. Either of you will have to merge the changes.